### PR TITLE
refactor(ramp): update ramp copy

### PR DESCRIPTION
**Pull request by wachunei**
## **Description**

This PR contains the changes requested to comply with some regulations

## **Related issues**

Fixes:

## **Manual testing steps**

1. Navigate through the changed screens and see the new copy

## **Screenshots/Recordings**

| Image |
|:-----:|
|  |
|  |
|  |
|  |
|  |
|  |

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by sethkfman — DISMISSED** [review]
LGTM

**Comment by wachunei** [comment]
@MetaMask/swaps-engineers, @MetaMask/confirmations, @MetaMask/metamask-assets I need your review please 🙏

**Review by smilingkylan — DISMISSED** [review]
LGTM

**Review by EtherWizard33 — APPROVED** [review]

File: locales/languages/en.json [hunk 1]
```diff
@@ -1389,6 +1392,10 @@
     "insufficient_amount": "You need {{amount}} more {{tokenSymbol}} to complete this transaction.",
     "buy_more_eth": "Buy more ETH",
     "buy_more": "Buy more",
+    "more_to_continue": "More {{ticker}} is needed to continue",
+    "you_can_also_send_funds": "You can also send funds from another account.",
+    "token_marketplace": "Token marketplace",
```
Comment (jclancy93) [inline locales/languages/en.json:1397]: Do we need two instances of the same string with different capitalizations?

**Review by jclancy93 — APPROVED** [review]

File: locales/languages/en.json (hunk 1 above)
Comment (wachunei) [reply locales/languages/en.json:1397]: Yes, they are used in different contexts according to designs.

File: app/components/Views/confirmations/SendFlow/Amount/index.js [hunk 2]
```diff
@@ -1346,11 +1347,20 @@ class Amount extends PureComponent {
...
+                  {strings('transaction.more_to_continue', {
+                    ticker: getTicker(ticker),
+                  })}
+                  {'\n'}
+                  <Text style={[styles.error, styles.underline]}>
+                    {strings('transaction.token_Marketplace')}
+                  </Text>
+                  {'\n'}
+                  {strings('transaction.you_can_also_send_funds')}
                 </Text>
```
Comment (jpuri) [inline app/components/Views/confirmations/SendFlow/Amount/index.js:1361]: I think we should be able to get new line by properly styling `Text` components and `\n` should not be needed for this purpose.

Comment (wachunei) [reply app/components/Views/confirmations/SendFlow/Amount/index.js:1361]: @jpuri In this case a new line is part of the text itself and not part of a layout/style.

**Review by gambinish — APPROVED** [review]

File: app/components/Views/confirmations/SendFlow/SendTo/index.js [hunk 3]
```diff
@@ -330,10 +330,13 @@ class SendFlow extends PureComponent {
 
     return (
       <>
-        <Text bold style={styles.buyEth} onPress={this.goToBuy}>
-          {strings('fiat_on_ramp.buy', {
-            ticker: getTicker(this.props.ticker),
-          })}
+        <Text> </Text>
```
Comment (gambinish) [inline app/components/Views/confirmations/SendFlow/SendTo/index.js:333]: Empty `Text` component, is this intentional?

File: app/components/Views/confirmations/SendFlow/SendTo/index.js [hunk 4]
```diff
@@ -330,10 +330,13 @@ class SendFlow extends PureComponent {
...
+        <Text reset bold link underline onPress={this.goToBuy}>
+          {strings('fiat_on_ramp_aggregator.token_marketplace')}.
+        </Text>
+        <Text reset>
+          {'\n'}
```
Comment (gambinish) [inline app/components/Views/confirmations/SendFlow/SendTo/index.js:338]: Can we use css here to space out the `Text` components within it's parent?

File: app/components/Views/confirmations/SendFlow/SendTo/index.js (hunk 3 above)
Comment (wachunei) [reply app/components/Views/confirmations/SendFlow/SendTo/index.js:333]: Yes, the text is appended to an existing text so it needs a space at the start.

**Review by gambinish — APPROVED** [review]

**Review by tommasini — APPROVED** [review]

**Pull request merged**